.github/workflows/tester.yaml

```
@@ -25,6 +24,7 @@ jobs:
         uses: home-assistant/actions/helpers/info@master
         with:
           path: ./${{ matrix.addon }}
+          
```

yamllint

[error] trailing spaces

(trailing-spaces)